### PR TITLE
Implement drop down menu for start/end coverage in "image"

### LINKED ISSUE #29: Make drop down menu for start/end coverage in "image"
with fields like: PMJ, C2, C3, etc.

### PR DESCRIPTION
**Pull request by alexfoias**
In order to implement the dropdown list, we have to do the following steps:
- Modify the repo on the local station (i.e. tristano)
- Replace all the start_coverage & end_coverage which are defined isnull in the tables  with empty field, associated with N/A label. This can be done using the commands:
```
Image.objects.filter(start_coverage__isnull=True).update(start_coverage='')
Image.objects.filter(end_coverage__isnull=True).update(end_coverage='')
```

Fixes #29

### CODE REVIEW

**Comment by jcohenadad** [comment]
thanks @alexfoias, i will look into it, just for the future, could we pls avoid to only mention issue title in PR title (see https://github.com/neuropoly/spinalcordtoolbox/blob/master/CONTRIBUTING.rst#id41). Thanks

File: sct_annotation/annotations/models.py [hunk 1]
```diff
@@ -93,6 +93,20 @@ def __str__(self):
...
+        ('PMJ','PMJ'),
+        ('C1','C1'),
+        ('C2','C2'),
+        ('C3','C3'),
+        ('C4','C4'),
+        ('C5','C5'),
+        ('C6','C6'),
+        ('C6','C6'),
+        ('C7','C6'),
+        ('C8','C8'),
```
Comment (jcohenadad) [inline sct_annotation/annotations/models.py]: C8 is not a vertebral level (stops at C7 in humans, then it is T1, T2, etc.)

File: sct_annotation/annotations/models.py [hunk 2]
```diff
@@ -93,6 +93,20 @@ def __str__(self):
...
+        ('','N/A'),
+        )
```
Comment (jcohenadad) [inline sct_annotation/annotations/models.py]: didn't we want to have more coverage? T1, T2, etc. until L3?

Comment (alexfoias) [reply sct_annotation/annotations/models.py]: I forgot to push the commits to the branch. Here is the full list included in the [commit](https://github.com/neuropoly/sct_testing_management/pull/61/commits/973d41cef88e88274ab548d18793059b466b2f35)

**Comment by alexfoias** [comment]
@jcohenadad Sorry for not following the template for this PR. I will follow the guidelines for all my future PR, branches and commits.

**Pull request merged**